frontend/packages/dev-console/integration-tests/features/addFlow/create-from-git.feature

```diff
@@ -172,3 +172,31 @@ Feature: Create Application from git form
                   | https://github.com/sclorg/django-ex.git        | django-ex-git-app  | django-ex-git         |
                   | https://github.com/spring-projects/spring-boot | spring-bottom-app  | openshift-quickstarts |
                   | https://github.com/sclorg/nodejs-ex.git        | nodejs-ex-git-app  | nodejs-ex-git         |
+
+
+        @regression @manual
+        Scenario Outline: Dotnet Builder iamge detection for git url "<git_url>": A-06-TC13
+            Given user is at Import from git page
+             When user enters Git Repo url as "<git_url>"
+              And user selects Show advanced Git options
+              And user enters Context dir as "<dir_name>"
+             Then git url gets Validated
+              And builder image is detected
+
+        Examples:
+                  | git_url                                                   | dir_name | app_name              | name              |
+                  | https://github.com/redhat-developer/s2i-dotnetcore-ex.git | /app     | dotnetcore-ex-git-app | dotnetcore-ex-git |
+
+
+        @regression @manual
+        Scenario Outline: Select Builder iamge manually for server related git urls": A-06-TC14
+            Given user is at Import from git page
+             When user enters Git Repo url as "<git_url>"
```

Comment:
Could we add a step here that the user will be able to see `Unable to detect the Builder Image.`?

Reply:
It makes sense to add the above scenario for httpd/nginx, as we will not able to detect it. Definitely gives clarity to the whoever tests it manually.

Reply:
Its a verification step, so i added it in Then statement ([Line no. 197](https://github.com/openshift/console/pull/9493/files/02cfb6419a107ec5a4c9417eafcf96ddd7a93063#diff-ee94359b91abeaf51a2ff23231eadc1db082bae01cc00c3c850f7ff39cf7f260R197)) instead of adding it in "When" statement